src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php

```diff
@@ -81,6 +81,8 @@ public function getVersionAction(Request $request)
             return JsonResponse::create('', JsonResponse::HTTP_NO_CONTENT);
         }
 
-        return JsonResponse::fromJsonString($hubResponse->getBody()->getContents());
+        $hubResponse = json_decode($hubResponse->getBody()->getContents(), true);
+
+        return new JsonResponse($hubResponse);
```

Comment:
Looks like it's related to encoding options (Symfony uses `JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_AMP | JSON_HEX_QUOT`).